projects/plugins/jetpack/class.jetpack-plan.php

```diff
@@ -103,7 +103,6 @@ class Jetpack_Plan {
 				'ecommerce-bundle',
 				'ecommerce-bundle-monthly',
 				'ecommerce-bundle-2y',
-				'vip',
```

Comment:
By moving it over, will that affect sites still on "professional" plans?

Reply:
Yes, but in terms of Jetpack proper, the same features are included in the two plans. (Complete comes with CRM but that's handled outside of Jetpack)

Reply:
So if I'm understanding correctly, for the sites that are still on the "professional plan", this will not affect any current functionality then on those sites?

Reply:
Yup, everything will work the same for them.

Reply:
Wanted to double-confirm this.

An existing site would have been provisioned with the `professional` plan, and will soon be switched over to reading from the new `VIP_JETPACK_DEFAULT_PLAN` constant as the plan option (seen in the other PR).

We can expect not problems for those existing sites, even though they weren't provisioned under "complete"?

Reply:
No, this is fine. Both the VIP plan slug is the important bit within Jetpack and Professional vs Complete doesn't matter within Jetpack itself as the feature sets are identical.

The plan differences are mainly around things like CRM and Jetpack Backups, which either require a license key into the CRM plugin or things that VIP doesn't use.

Reply:
I was a bit confused with this as well; since the feature set is the same for both Professional and complete, what's the benefit of switching from professional to complete here now? If we left `vip` where it was until today, wouldn't the vip plan then be a match for both Professional and Complete? Wouldn't that be preferable?